tests/cpp/unit-tests/test_ssdb.cpp

```diff
@@ -0,0 +1,63 @@
+#include "../../../third-party/catch/catch.hpp"
+#include "redis.h"
+
+using namespace SmartRedis;
+
+// Helper class used for accessing protected members of RedisServer
+class TestSSDB : public Redis
+{
+    public:
+        TestSSDB() : Redis() {}
+
+        std::string get_ssdb()
+        {
+            return this->_get_ssdb();
+        }
+};
+
+// helper function for putting the SSDB environment
+// variable back to its original state
+void putenv_ssdb(const char* ssdb)
+{
+    if (ssdb != nullptr) {
+        std::string reset_ssdb = std::string("SSDB=") + std::string(ssdb);
+        char* reset_ssdb_c = new char[reset_ssdb.size() + 1];
+        std::copy(reset_ssdb.begin(), reset_ssdb.end(), reset_ssdb_c);
+        reset_ssdb_c[reset_ssdb.size()] = '\0';
+        putenv( reset_ssdb_c);
```

Comment:
Can we just use ``reset_ssdb.c_str()`` to avoid all of the ``char*`` memory work?

Reply:
As we discussed and found out together, `std::string::c_str()` returns `const char*` and `putenv` has the signature `int putenv(char*)`, so we will keep the code as is in order to avoid casting from `const char*` to `char*`